internal/cmd/gocdk/internal/static/_assets/biome/versions.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
```

Comment:
This should go in main.tf, not a new file.

Reply:
OK. The upgrade process created a new file, but I missed that we already had a version section in `main.tf`.

Done.